src/e_ibmca.c

```diff
@@ -1527,6 +1528,7 @@ typedef unsigned int (*ica_aes_gcm_last_t)(unsigned char *icb,
 					   unsigned int direction);
 
 /* entry points into libica, filled out at DSO load time */
+ica_set_fallback_mode_t		p_ica_set_fallback_mode;
```

Comment:
please align p_ica_set_fallback_mode with the below variables.

Reply:
ok, will do